packages/palette/src/Theme.tsx

```diff
@@ -322,7 +322,7 @@ export const themeProps = {
 }
 
 /**
- * Creates a new Grid context for web. On ReactNative it serves as a noop.
+ * Creates a new Grid context for web. On React Native it serves as a noop.
```

Comment:
@zephraph - have you ever encountered babel or npm or whatever corrupting a version? I just spent a good amount of time trying to figure out why all the tests in reaction broke due to something that shouldn't have had any effect and had a hunch that it was some kind of high weirdness... sure enough, release a new version with a new whitespace character and everything was all good. 

It was even weirder. When linking locally, if i referenced master (broken) and then did a clean and then jumped back to a previous has (good version) it would still break. It was like something in the broken version contaminated yarn or node or something. It was really odd.